eleventy.config.js

```diff
@@ -67,6 +67,30 @@ module.exports = function (eleventyConfig) {
     return collection.getFilteredByGlob('app/posts/bowel-screening/**/*.md')
   })
 
+  eleventyConfig.addCollection('record-a-vaccination', collection => {
```

Comment:
Suggest we divide these with comments for each area - Screening, vaccinations, personalised prevention.

Reply:
Good shout. Done.